workshops/templates/workshops/workshop_staff.html

```diff
@@ -43,6 +43,7 @@
       </tr>
     {% endfor %}
     </table>
+    <p><a href="mailto:{{ emails|join:',' }}" class="btn btn-primary">EMail All</a></p>
```

Comment:
You can suggest a better caption for the button. Couldn't come up with anything better.